cmd/maya-apiserver/app/server/pool_endpoint_v1alpha1.go

```diff
@@ -62,9 +62,8 @@ func (p *poolAPIOpsV1alpha1) httpGet() (interface{}, error) {
 	return p.read(poolName)
 }
 
-func (p *poolAPIOpsV1alpha1) list() (*v1alpha1.StoragePoolList, error) {
-	glog.Infof("received storage pool list request")
-
+func (p *poolAPIOpsV1alpha1) list() (*v1alpha1.CStorPoolList, error) {
+	glog.Infof("CAS template based storage pool list request was received")
```

Comment:
it seems - logs can be removed from here and at the end as well, and in next function also.

Reply:
fixed